src/main/java/com/spotinst/sdkjava/example/elastigroup/aws/StatefulElastigroupUsageExample.java

```diff
@@ -0,0 +1,109 @@
+        Boolean resumeStatus = resumeStatefulInstance(elastigroupClient, ELASTIGROUP_ID, statefulId);
+
+        //Recycle the stateful instance
+        System.out.println("----------Recycle stateful Instance--------------");
+        Boolean recycleStatus = recycleStatefulInstance(elastigroupClient, ELASTIGROUP_ID, statefulId);
+
+        //Deallocate the stateful instance
+        System.out.println("----------Deallocate stateful Instance--------------");
+        Boolean deallocateStatus = deallocateStatefulInstance(elastigroupClient, ELASTIGROUP_ID, statefulId);
+
+    }
+
+    private static Boolean pauseStatefulInstance(SpotinstElastigroupClient client, String elastigroupId, String statefulId) {
+
+        Boolean pauseStatus = client.pauseStatefulInstance(elastigroupId, statefulId);
+
+        if (pauseStatus) {
+            System.out.println("Pause operation is started successfully");
+        }
+        return pauseStatus;
+    }
+
+    private static Boolean resumeStatefulInstance(SpotinstElastigroupClient client, String elastigroupId, String statefulId) {
+
+        Boolean resumeStatus = client.resumeStatefulInstance(elastigroupId, statefulId);
+
+        if (resumeStatus) {
+            System.out.println("Resume operation is started successfully");
+        }
+        return resumeStatus;
+    }
+
+    private static Boolean recycleStatefulInstance(SpotinstElastigroupClient client, String elastigroupId, String statefulId) {
+
+        Boolean recycleStatus = client.pauseStatefulInstance(elastigroupId, statefulId);
+
+        if (recycleStatus) {
+            System.out.println("Recycle operation is started successfully");
+        }
+        return recycleStatus;
+    }
+
+    private static Boolean deallocateStatefulInstance(SpotinstElastigroupClient client, String elastigroupId, String statefulId) {
+
+        Boolean deallocateStatus = client.deallocateStatefulInstance(elastigroupId, statefulId);
+
+        if (deallocateStatus) {
+            System.out.println("Deallocate operation is started successfully");
+        }
+        return deallocateStatus;
+    }
+
+    private static List<ElastigroupListStatefulInstancesResponse> listStatefulInstances(SpotinstElastigroupClient client, String elastigroupId) {
+
+        List<ElastigroupListStatefulInstancesResponse> elastigroupStatefulInstances =
+                client.listStatefulInstances(elastigroupId);
+
+        for (ElastigroupListStatefulInstancesResponse statusData : elastigroupStatefulInstances) {
+            System.out.println(String.format("Stateful Id: %s", statusData.getId()));
+            System.out.println(String.format("Stateful Instance Id: %s", statusData.getInstanceId()));
+            System.out.println(String.format("Stateful Instance Image Id: %s", statusData.getImageId()));
+            System.out.println(String.format("Stateful Instance private IP: %s", statusData.getPrivateIp()));
```

Comment:
Print all parameters. Especially interested to see if createdAt and LaunchedAt parameters are correctly returned or not.